artifactory/commands/generic/search.go

```diff
@@ -11,12 +11,14 @@ import (
 
```

Comment:
To remain consistent with the upload and download commands, I think that in the includeDirs property should also be added to a File Spec.